src/main/java/jenkins/plugins/logstash/configuration/Logzio.java

```diff
@@ -0,0 +1,110 @@
+package jenkins.plugins.logstash.configuration;
+
+import hudson.Extension;
+import hudson.util.FormValidation;
+import hudson.util.Secret;
+
+import jenkins.plugins.logstash.persistence.LogzioDao;
+import jenkins.plugins.logstash.Messages;
+
+import org.apache.commons.lang.StringUtils;
+import org.kohsuke.stapler.DataBoundConstructor;
+import org.kohsuke.stapler.DataBoundSetter;
+import org.kohsuke.stapler.QueryParameter;
+
+import javax.annotation.Nonnull;
+
+public class Logzio extends LogstashIndexer<LogzioDao>
+{
+    private Secret key;
```

Comment:
The key is basically passed through to `HttpsRequestConfiguration` from logz.io in the method `setLogzioToken`.
So wouldn't it make sense to name the field and the methods accordingly as 'token'?